internal/uidriver/glfw/ui_unix.go

```diff
@@ -30,7 +30,12 @@ import (
 
 type videoModeScaleCacheKey struct{ X, Y int }
 
-var videoModeScaleCache map[videoModeScaleCacheKey]float64
+var videoModeScaleCache = map[videoModeScaleCacheKey]float64{}
+
+// updateMonitorsByOS must be called from the main thread.
+func updateMonitorsByOS() {
+	videoModeScaleCache = map[videoModeScaleCacheKey]float64{}
```

Comment:
In general, just a for-loop is more efficient to clear a map:

```
for k := range videModeScaleCache {
    delete(videModeScaleCache, k)
}
```

This should be optimized by Go compiler.